code/modules/html_interface/voting/voting.dm

```diff
@@ -271,6 +275,11 @@ var/global/datum/controller/vote/vote = new()
 	return 0
 
 /datum/controller/vote/proc/initiate_vote(var/vote_type, var/initiator_key, var/popup = 0, var/weighted_vote = 0)
+	if(currently_voting)
+		for(var/client/C in admins)
+			to_chat(C, "<span class='info'>[C.ckey] attempted to begin a vote, however a vote is already in progress.</span>")
```

Comment:
use message_admins()

Reply:
ill do it when i get back

Reply:
it does "ADMIN LOG: Goldsteel attempted to begin a vote, however a vote is already in progress." now and seems fine